pkg/cmdutil/userio/wizard/wizard.go

```diff
@@ -177,21 +198,18 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		return m, tea.Sequence(updateListener, cmd)
 	case tea.KeyMsg:
 		log.Debug().Msgf("Wizard: Received keystroke [%s]", msg.String())
-
 		switch msg.Type {
 		case tea.KeyCtrlC:
```

Comment:
This will pass keystroke to the input model only in ctrl+c is entered, won't it? Or am I missing something?

Reply:
Ah, I didn't notice you pass every message to the inner model a bit later, sorry